exporter/context.go

```diff
@@ -205,7 +253,24 @@ func (ic *importContext) Run() error {
 	if len(ic.Scope) == 0 {
 		return fmt.Errorf("no resources to import")
 	}
-	sh, err := os.OpenFile(fmt.Sprintf("%s/import.sh", ic.Directory), os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0755)
+	shFileName := fmt.Sprintf("%s/import.sh", ic.Directory)
+	if ic.incremental {
+		shFile, err := os.Open(shFileName)
+		if err == nil {
+			fileScanner := bufio.NewScanner(shFile)
+			fileScanner.Split(bufio.ScanLines)
+			for fileScanner.Scan() {
+				line := fileScanner.Text()
+				if strings.HasPrefix(line, "terraform import ") {
+					ic.shImports[strings.TrimRight(line, "\n")] = true
+				}
+			}
+			shFile.Close()
```

Comment:
Not closed in the err path (may be problematic on Windows).

```go
defer shFile.close()
```

Reply:
but `shFile` will be non-nil only if `err == nil`